b2luigi/core/utils.py

```diff
@@ -238,16 +239,20 @@ def get_log_file_dir(task):
     if hasattr(task, 'get_log_file_dir'):
         log_file_dir = task.get_log_file_dir()
         return log_file_dir
-        
+
     # Be sure to evaluate things relative to the current executed file, not to where we are now
-    base_log_file_dir = map_folder(get_setting("log_dir", task=task, default="logs", 
+    base_log_file_dir = map_folder(get_setting("log_dir", task=task, default="logs",
                                                deprecated_keys=["log_folder"]))
     log_file_dir = create_output_file_name(task, task.get_task_family() + "/", result_dir=base_log_file_dir)
 
     return log_file_dir
 
 
 def get_task_file_dir(task):
+    if hasattr(task, 'get_task_file_dir'):
```

Comment:
Yes, I am fine with that!

I thought about also having a line like `base_dir = ...` as in the `get_log_file_dir` function, but that would duplicate code.
So lets go with that!